docs/how-to/custom-base-templates.md

```diff
@@ -0,0 +1,161 @@
+# Use custom base HTML templates
+
+If the default template in the Starter Pack doesn't fully meet your needs -- whether you want a unique layout, a custom header or footer, or a specialized sidebar for certain pages -- you can create and use a custom base HTML template for your Sphinx project.
+
+This guide shows you how to extend or override the default templates to tailor the look and structure of your documentation.
+
+```{note}
+Base template customizations can be made to your documentation.
+However, they are not officially supported by the team maintaining the starter pack.
+Use them at your own discretion.
+```
```

Comment:
I think I'd exclude this part. To me, it sounds like an unnecessary caution, which users might find discouraging compared to our other pages.